exampleCourse/courseInstances/SectionA/assessments/demo/externalGrading/infoAssessment.json

```diff
@@ -17,6 +17,7 @@
                 {"id": "demo/autograder/python/plots",          "points": 1, "maxPoints": 5},
                 {"id": "demo/autograder/python/random",         "points": 1, "maxPoints": 5},
                 {"id": "demo/autograder/python/earlyExit",      "points": 1, "maxPoints": 5},
+                {"id": "demo/autograder/python/randomFunctionName",      "points": 1, "maxPoints": 5},
```

Comment:
Complete nit, and not requesting a change because things were already inconsistent before this PR: IMO we should either commit to careful manual formatting/alignment or just let Prettier take over.

Reply:
Agree here. Is there JSON validation as part of the current CI? If not, I would support some type of JSON linting, and using Prettier for all of the JSON files.

Reply:
JSON /validation/ happens implicitly (because we sync the example course), but autoformatting via Prettier is not currently done.